### ISSUE TITLE
Feature Request: Create/Edit Intel

### ISSUE THREAD

**Issue by luke-metcalf** [zen-mod/ZEN#29]
There should be an ability to create intel where it will spawn in a type of intel to collect such as a laptop, notes, documents etc. Moreover, editing an object already to have intel such as a car or a device. 
![CreateEdit Intel](https://user-images.githubusercontent.com/23123952/60385032-a8cadc80-9a7c-11e9-9ed7-918f9e6217ad.jpg)

**Comment by oOKexOo** [comment]
Side note: Could make use of icons like for the doors, which allows locating/editing the intel more easily.

**Comment by mharis001** [comment]
Adding clickable icons for this I think would create an unnecessary performance impact as the positions /size of the icons has to be updated every frame. I think this is best kept as a simple feature accessible through the module.

**Comment by jonpas** [comment]
ACE Interact for Intel object when ACE is running llease. :)

**Comment by mharis001** [comment]
Yeah, that would be good. One issue is what to do with the "Action Duration" option:
 
1. Use a progress bar
2. Ignore it for ACE Interact Menu action
3. Remove it completely

I am leaning towards option 3 because I don't think its necessary but it may be useful for some users.

**Comment by jonpas** [comment]
3, alternatively 1.

**Comment by CreepPork** [comment]
Action duration should be kept but then the first option. Because you can use the intel module to create custom actions e.g. Hack this Terminal.

**Comment by CreepPork** [comment]
As for the icons, it could be a CBA setting to draw the icons. I think that it would be quite a useful feature to be able to locate your intel object in the sea of objects you have around you quickly.

**Comment by jonpas** [comment]
> As for the icons, it could be a CBA setting to draw the icons. I think that it would be quite a useful feature to be able to locate your intel object in the sea of objects you have around you quickly. 

ACE Interaction solves that already. No clue how vanilla "hold action" works.

**Comment by CreepPork** [comment]
No, it's about the Zeus interface.

**Comment by CreepPork** [comment]
Drawing the icons for the players would be horrendous and would degrade the point of intel.

**Comment by CreepPork** [comment]
> Yeah, that would be good. One issue is what to do with the "Action Duration" option:
> 
> 1. Use a progress bar
> 2. Ignore it for ACE Interact Menu action
> 3. Remove it completely
> 
> I am leaning towards option 3 because I don't think its necessary but it may be useful for some users.

As I'm thinking about this more, the first option should only be used for the ACE interaction as it has no "hold down the key until completed" behaviour if I recall correctly. However, it should not be used for the vanilla hold down key as it already implements this behaviour.

**Comment by oOKexOo** [comment]
Alternatively to an icon, we could just attach the module to the object and make it editable, like the "Neutralize" objective module does.

**Comment by CreepPork** [comment]
But then you would have a harder time adjusting the item because of the overlay.

**Comment by neilzar** [comment]
To continue the discussion, I agree with the following:

> As I'm thinking about this more, the first option should only be used for the ACE interaction as it has no "hold down the key until completed" behaviour if I recall correctly. However, it should not be used for the vanilla hold down key as it already implements this behaviour.

As for the separate module, it would be useful because there are many times where I would delete intel because I made a mistake. A module would be an easy solution to this, but I can think of another way; similar to how currently documents can be edited. You place the object and it immediately comes up with the edit dialog, and you can double click the object to re-open it.

![image](https://user-images.githubusercontent.com/3193528/63726166-bf16ce00-c85c-11e9-8c70-80b21b0d78b8.png)
